### PR TITLE
[Azure Search] Add SDK bindings for L-series SKUS

### PR DESCRIPTION
**Pull request by smaine**
This PR enables the .NET SDK to create new instances of the Azure Search L-Series SKU.

### CODE REVIEW

File: src/SDKs/Search/Management/Management.Search/Generated/ServicesOperations.cs [hunk 1]
```diff
@@ -935,6 +935,202 @@ public async Task<AzureOperationResponse<IEnumerable<SearchService>>> ListByReso
...
+        /// <exception cref="ValidationException">
+        /// Thrown when a required parameter is null
+        /// </exception>
+        /// <exception cref="System.ArgumentNullException">
+        /// Thrown when a required parameter is null
+        /// </exception>
+        /// <return>
+        /// A response object containing the response body and response headers.
+        /// </return>
+        public async Task<AzureOperationResponse<IEnumerable<SearchService>>> ListBySubscriptionWithHttpMessagesAsync(SearchManagementRequestOptions searchManagementRequestOptions = default(SearchManagementRequestOptions), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```
Comment (brjohnstmsft) [inline src/SDKs/Search/Management/Management.Search/Generated/ServicesOperations.cs]: Looks like we're a bit behind on updating the management SDK from the spec. This was added to the spec about a month ago. It's going to need test coverage (we can sync offline about who should do that work).

**Review by bitsforthought — APPROVED** [review]
We need to find the right person to add test coverage for the new "ListBySubscription" API, but that can be a separate PR. Your changes LGTM.

**Comment by dsgouda** [comment]
@brjohnstmsft is this ready to be merged?

**Review by dsgouda — APPROVED** [review]
LGTM
Package `3.0.0` is still not published so these should be fine.

**Comment by brjohnstmsft** [comment]
@dsgouda This is fine to merge. We still want to add a bit of test coverage for an unrelated change before pushing 3.0 to NuGet, but that will be a separate PR.

**Pull request merged**